week-5/diego_jonathan_deyvi/ext_queries.sql

```diff
@@ -0,0 +1,39 @@
+--Query 5
+select pur.pos, COUNT(*), SUM(pro.p_price)
+FROM purchase as pur
+  INNER JOIN product as pro on pro.id = pur.product_id
+GROUP BY pur.pos
+ORDER BY sum DESC
+LIMIT 1;
+
+--Query 6
+(SELECT pro
```

Comment:
I like how you used UNION, but this line break makes it difficult to understand, I would recommend you to have an alias of one word and also to name the field without mentioning the table, it would look like this:
```sql
(SELECT p.name, . . . .
```

Reply:
Nice recommendation, I'll use in my following queries 👍